content/200-concepts/100-components/03-prisma-migrate/050-prisma-migrate-flows.mdx

```diff
@@ -0,0 +1,234 @@
+---
+title: 'Migration flows'
+metaTitle: 'Migration flows (Concepts)'
+metaDescription: 'Prisma Migrate is a declarative data modeling and schema migration tool that is available via the Prisma CLI.'
+---
+
+<TopBlock>
+
+</TopBlock>
+
+## Modifying a migration
+
+You can choose to change the behavior of a migration **when you create it**. For example, Prisma Migrate cannot automatically detect that you want to _rename_ a field - instead, it will add a new field and remove the old one. To rename the field instead (for example, `name` &rarr; `fullname`), you must create and modify a **draft migration**:
+
+1. Make a change to your schema - for example, renaming a field:
+ 
+    ```prisma file=schema.prisma
+    model User {
+    +  fullname String?
+    -  name     String?
+    }
+    ```
+
+1. Use the `--create-only` option to generate a **draft** migration:
+
+    <CodeWithResult>
+    <Cmd>
+
+    ```terminal
+    prisma migrate dev --create-only --preview-feature
+    ```
+    </Cmd>
+    <CmdResult>
+    ```
+    Prisma Migrate created the following migration without applying it 20201208100950_fullname_field
+    ```    
+    </CmdResult>
+    </CodeWithResult>
+
+    Prisma Migrate does not apply this migration.
+
+2. Edit the generated `20201208100950_fullname_field/migration.sql` file:
+
+   ```sql
+   ALTER TABLE "User" DROP COLUMN "name",
+   - ADD COLUMN     "fullname" TEXT,
+   + RENAME "name" TO  "fullname"
+   ```
+
+3. Migrate:
+
+   ```terminal
+   prisma migrate dev --preview-feature
+   ```
+
+## Deploying migrations in a non-development environment
+
+Use the following command in non-development environments to apply migrations:
+
+```terminal
+prisma migrate deploy --preview-feature
+```
+
+The `prisma migrate dev` command is reserved for development environments and requires the [shadow database](../#shadow-database), which is not available in production.
+
+## Collaborating with a team
+
+To bring in changes to the database schema and migrations from collaborators, pull the changed Prisma Schema and the new migration files. Run `prisma migrate dev` to apply the changes. Running this command:
+
+1. Checks if there are any migrations that have not yet been applied and applies those migrations
+1. Checks for changes in the Prisma schema - if there are changes, Prisma creates and applies new migrations
+
```

Comment:
Second should be 2.
And also I think it should say if there are changes that are not reflected in the now also potentially changed migration history it creates and applies new migrations.